go/vt/vttablet/tabletserver/throttle/throttler_test.go

```diff
@@ -803,38 +840,99 @@ func TestIsAppThrottled(t *testing.T) {
-		assert.False(t, throttler.IsAppThrottled("app1")) // exempted
-		assert.False(t, throttler.IsAppThrottled("app2")) // expired
-		assert.True(t, throttler.IsAppThrottled("app3"))
-		assert.False(t, throttler.IsAppThrottled("app4"))      // ratio is zero
-		assert.False(t, throttler.IsAppThrottled("app_other")) // not specified
+		{
+			throttled, app := throttler.IsAppThrottled("app1")
+			assert.False(t, throttled) // exempted
+			assert.Equal(t, "app1", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app2")
+			assert.False(t, throttled) // expired
+			assert.Equal(t, "app2", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app3")
+			assert.True(t, throttled)
+			assert.Equal(t, "app3", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app4")
+			assert.False(t, throttled) // ratio is zero
+			assert.Equal(t, "app4", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app_other")
+			assert.False(t, throttled) // not specified
+			assert.Equal(t, "app_other", app)
+		}
 
 		assert.Equal(t, 3, throttler.throttledApps.ItemCount())
 	})
 	t.Run("all", func(t *testing.T) {
 		// throttle "all", see how it affects app
 		throttler.ThrottleApp(throttlerapp.AllName.String(), plusOneHour, DefaultThrottleRatio, false)
 		defer throttler.UnthrottleApp(throttlerapp.AllName.String())
-		assert.True(t, throttler.IsAppThrottled("all"))   //
-		assert.False(t, throttler.IsAppThrottled("app1")) // exempted
-		assert.True(t, throttler.IsAppThrottled("app2"))  // expired, so falls under "all"
-		assert.True(t, throttler.IsAppThrottled("app3"))
-		assert.False(t, throttler.IsAppThrottled("app4"))     // ratio is zero, there is a specific instruction for this app, so it doesn't fall under "all"
-		assert.True(t, throttler.IsAppThrottled("app_other")) // falls under "all"
+		{
+			throttled, app := throttler.IsAppThrottled("all")
+			assert.True(t, throttled) // explicitly throttled
+			assert.Equal(t, "all", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app1")
+			assert.False(t, throttled) // exempted
+			assert.Equal(t, "app1", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app2")
+			assert.True(t, throttled) // expired, so falls under "all"
+			assert.Equal(t, "all", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app3")
+			assert.True(t, throttled)
+			assert.Equal(t, "app3", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app4")
+			assert.False(t, throttled) // ratio is zero, there is a specific instruction for this app, so it doesn't fall under "all"
+			assert.Equal(t, "app4", app)
+		}
+		{
+			throttled, app := throttler.IsAppThrottled("app_other")
+			assert.True(t, throttled) // falls under "all"
+			assert.Equal(t, "all", app)
+		}
 
 		// continuing previous test, we had 3 throttled apps. "all" is a new app being throttled.
 		assert.Equal(t, 4, throttler.throttledApps.ItemCount())
 	})
-	//
+	// 	//
```

Comment:
Looks like we added an extraneous comment indicator.

Reply:
removed